src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -210,14 +210,12 @@ public function cloneVar($var, $filter = 0)
         $this->filter = $filter;
 
         try {
+            gc_disable();
```

Comment:
shown also in linked profiles: cloning involves lots of `Stub` objects, thus can trigger lots of GC calls, for no benefit.